lua/graph.lua

```diff
@@ -878,7 +895,7 @@ function filter_tags_generic(kv)
     kv["emergency_backward"] = kv["emergency_forward"]
 
     if (kv["bike_backward"] == "false" and kv["oneway:bicycle"] ~= "-1" and 
-       (kv["oneway:bicycle"] == nil or oneway[kv["oneway:bicycle"]] == false)) then
+       (kv["oneway:bicycle"] == nil or oneway[kv["oneway:bicycle"]] == false or kv["oneway:bicycle"] == "no")) then
```

Comment:
Before
![screenshot from 2017-07-20 16-07-55](https://user-images.githubusercontent.com/28691264/28436672-cb123694-6d65-11e7-86b6-0f35db1350ed.png)

After
![screenshot from 2017-07-20 16-08-17](https://user-images.githubusercontent.com/28691264/28436686-d6c28e08-6d65-11e7-9e8e-26912ae3e6e4.png)

These are the tags that are on that road
![screenshot from 2017-07-20 16-08-37](https://user-images.githubusercontent.com/28691264/28436696-def630d4-6d65-11e7-9a22-946c19856edc.png)